client/src/app/shared/directives/not-found-description.directive.ts

```diff
@@ -0,0 +1,4 @@
+import { Directive } from '@angular/core';
+
+@Directive({ selector: '[notFoundDescription]' })
```

Comment:
`3:24  error  The selector should start with one of these prefixes: "os" (https://angular.io/guide/styleguide#style-02-08)  @angular-eslint/directive-selector`